Twig/Extension/TermTaxonomyExtension.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+namespace Ekino\WordpressBundle\Twig\Extension;
+
+use Ekino\WordpressBundle\Model\TermTaxonomy;
+use Ekino\WordpressBundle\Manager\OptionManager;
+
+class TermTaxonomyExtension extends \Twig_Extension
+{
+    /**
+     * @var OptionManager
+     */
+    protected $optionManager;
+
+    /**
+     * @param OptionManager $optionManager
+     */
+    public function __construct(OptionManager $optionManager)
+    {
+        $this->optionManager = $optionManager;
+    }
+
+    public function getFunctions()
+    {
+        return array(
+            new \Twig_SimpleFunction('wp_get_term_link', array($this, 'getTermLink')),
+        );
+    }
+
+    /**
+     * @param TermTaxonomy $termTaxonomy
+     * @param string $type The link type. Can be "category" or "tag"
+     *
+     * @return string
+     */
+    public function getTermLink(TermTaxonomy $termTaxonomy, $type = 'category')
+    {
+        $prefix = ($prefix = $this->optionManager->findOneByOptionName($type . '_base')) ? $prefix->getValue() : null;
+        $output = [$termTaxonomy->getTerm()->getSlug()];
```

Comment:
Can you use array() instead of [] since this bundle can be used since PHP 5.3.2, please?